galaxy_ng/app/settings.py

```diff
@@ -1,3 +1,16 @@
+import os
+
+DATABASES = {
```

Comment:
Curious, is this needed specifically for auth?

Reply:
Actually no, I will move it to a separate PR.
I'm trying to reduce number of custom configurations living in different places and put everything to `galaxy_ng/app/settings` to minimize custom settings size for different environments.